wiki/Beatmap/Circle_size/de.md

```diff
@@ -0,0 +1,39 @@
+---
+tags:
+  - CS
+  - key count
+  - keycount
+---
+
+# Circle Size
+
+*Für Vorschriften zur Circle Size, siehe: [Ranking-Kriterien](/wiki/Ranking_criteria)*
+
+Die **Circle Size** (***CS***) ist eine Schwierigkeitseinstellung einer [Beatmap](/wiki/Beatmap), die die Größe von [Hit-Objekten](/wiki/Gameplay/Hit_object) beeinflusst. Die Circle Size kann Werte von 0 bis 10 annehmen, der [Beatmap-Editor](/wiki/Client/Beatmap_editor) erlaubt aber nur Werte von 2 bis 7. Um andere Werte zu verwenden, muss die [`.osu`-Datei](/wiki/Client/File_formats/osu_(file_format)) der Beatmap manuell bearbeitet werden.
+
+## osu!
+
+In [osu!](/wiki/Game_mode/osu!) bestimmt die Circle Size die Größe der [Hit-Circles](/wiki/Gameplay/Hit_object/Hit_circle) und [Sliders](/wiki/Gameplay/Hit_object/Slider). Je höher der CS-Wert ist, desto kleiner sind die Hit-Objekte. Die [Spinner](/wiki/Gameplay/Hit_object/Spinner)-Größe wird nicht von der Circle Size beeinflusst. Die Größe der Hit-Circles wird durch folgende Formel bestimmt:
+
+`r = 54.4 - 4.48 * CS`
+
+Dabei ist `r` der Kreisradius in [osu!-Pixeln](/wiki/Client/Beatmap_editor/osu!_pixel) und `CS` die eingestellte Circle Size.
+
+## osu!taiko
+
+In [osu!taiko](/wiki/Game_mode/osu!taiko) hat die Circle Size keine Auswirkungen auf das Gameplay.
+
+## osu!catch
+
+In [osu!catch](/wiki/Game_mode/osu!catch) bestimmt die Circle Size die Größe des Catchers und der [Fruits](/wiki/Gameplay/Hit_object/Fruit).
```

Comment:
Nothing preventing us, but the existing German articles prefer the English words over translations (see: `Accuracy`, `Flashlight`, `Health`). There are a few examples of the translated terms being used, but it's less common.

Reply:
For what it's worth, I tried sticking to the original words when used in articles that are addressed to more experienced players. Articles that describe basic gameplay stuff use the German terms because I thought that in general, the English one is pretty common and beginners may prefer to learn the German equivalent first.

[Gameplay/Hit object](https://osu.ppy.sh/wiki/de/Gameplay/Hit_object) uses Fruit.

cc @respektive tagging you because I think we had this discussion too back then.

Reply:
If you're fine with both choices, could we maybe move this discussion to somewhere else? No need to block this PR for a relatively unrelated discussion.

Reply:
Can be discussed and adjusted any time anyway.